src/modules/launcher/Plugins/Microsoft.Plugin.Folder/Sources/Path/FolderHelper.cs

```diff
@@ -75,6 +76,17 @@ public static bool ValidDriveLetter(char c)
 
         public static string Expand(string search)
         {
+            if (search == null)
+            {
+                throw new ArgumentNullException(nameof(search));
+            }
+
+            // Absolute path of system drive: \Windows\System32
+            if (search.Length > 2 && search[0] == '\\' && search[1] != '\\')
+            {
```

Comment:
The above condition does not take into account the edge case. If the search query is `\` then it should interpret it as `C:\`.

Instead of the following
![image](https://user-images.githubusercontent.com/17161067/97896970-7dc25e00-1d3e-11eb-8ddc-f070ec77efbb.png)

We should get 

![image](https://user-images.githubusercontent.com/17161067/97897041-9894d280-1d3e-11eb-8f50-d5d2d68910fe.png)